packages/dxf-deserializer/index.js

```diff
@@ -575,7 +578,7 @@ const deserialize = function (src, filename, options) {
   const defaults = {
     version: '0.0.1',
     output: 'jscad',
-    strict: true,
+    strict: false,
```

Comment:
why the change to false ? Sorry If I am annoying with this, but parsers should ALWAYS throw : how that error gets propagated and handled is the job of the consumer (web/desktop app/cli /else), without an exception, there is no way of knowing what happened inside the de/serializers, and the app(s) could enter an unknown state